pkg/openai/client.go

```diff
@@ -236,15 +247,16 @@ func (c *Client) Call(ctx context.Context, messageRequest types.CompletionReques
 	}
 
 	request := openai.ChatCompletionRequest{
-		Model:       messageRequest.Model,
-		Messages:    msgs,
-		MaxTokens:   messageRequest.MaxTokens,
-		Temperature: messageRequest.Temperature,
-		Grammar:     messageRequest.Grammar,
```

Comment:
Was this an accidental deletion?

Reply:
That field does not exist in the package, only in our fork which we're migrating away from using